cdap-watchdog/src/test/java/co/cask/cdap/logging/framework/CDAPLogAppenderTest.java

```diff
@@ -213,6 +223,14 @@ public void testCDAPLogAppenderRotation() throws Exception {
       Assert.assertEquals(files.get(1).getEventTimeMs(), currentTimeMillisEvent1 + 1000);
       Assert.assertTrue(files.get(0).getFileCreationTimeMs() >= currentTimeMillisEvent1);
       Assert.assertTrue(files.get(1).getFileCreationTimeMs() >= currentTimeMillisEvent2);
+
+      // checking permission
+      String expectedPermissions = "rwxr-----";
```

Comment:
its only for tests. the default permission will be "600" if we didn't set any permissions.